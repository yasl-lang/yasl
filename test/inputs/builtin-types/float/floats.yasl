##10.0\n10.0\n0.11\n120.0\n13.53\n

echo 1.0e_1
echo 1.0e1
echo 1.1E-1
echo 1.2e+2
echo 11.0e-1 * 1.23E+1
