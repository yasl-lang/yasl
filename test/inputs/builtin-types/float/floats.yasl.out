10.0
10.0
0.11
120.0
13.53
